src/main/java/org/folio/search/service/setter/authority/NumberOfTitlesPostProcessor.java

```diff
@@ -0,0 +1,48 @@
+package org.folio.search.service.setter.authority;
+
+import static java.util.stream.Collectors.toList;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.IntStream;
+import lombok.RequiredArgsConstructor;
+import org.folio.search.client.EntitiesLinksClient;
+import org.folio.search.domain.dto.Authority;
+import org.folio.search.service.setter.PostProcessor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public final class NumberOfTitlesPostProcessor implements PostProcessor {
+
+  private final EntitiesLinksClient entitiesLinksClient;
+
+  @Override
+  public void process(List<Object> res) {
+    if (res == null || res.isEmpty() || !(res.get(0) instanceof Authority)) {
+      return;
+    }
+
```

Comment:
I suggest making PostProcessor generic by type of response object. 
In `ElasticsearchDocumentConverter` you can create a map of processors with a key by class. By doing so, you can get all processors for the exact object. This will decrease processing time for objects that are not require some postprocessing